operame.ino

```diff
@@ -337,6 +350,10 @@ void setup() {
     mqtt_template = WiFiSettings.string("operame_mqtt_template", "{} PPM", T.config_mqtt_template);
     WiFiSettings.info(T.config_template_info);
 
+    WiFiSettings.heading("sparkline");
+    sparkline_enable       = WiFiSettings.checkbox("operame_sparkline", false, T.config_sparkline);
+    sparkline_bufferlenght = WiFiSettings.integer("operame_sparkline_buffer", 0, 16383, 512, T.config_sparkline_buffer);    ; 
```

Comment:
Maybe the buffer length should not be user configurable, at least not without explaining what it does. If it is configurable, I think it would be slightly more user friendly to let them specify the buffer size in minutes.

Why 16383 and 512?